pepys_import/import.py

```diff
@@ -25,9 +25,17 @@ def main(path=DIRECTORY_PATH, db=DEFAULT_DATABASE):
     parser = argparse.ArgumentParser()
     path_help = "The path to import data from (The default value is the directory of the script)"
     db_help = "The database to connect to: (The is to use an in-memory database)"
+    archive_help = "The flag to move files after the import (The default value is True)"
     parser.add_argument(
         "--path", help=path_help, required=False, default=DIRECTORY_PATH
     )
     parser.add_argument("--db", help=db_help, required=False, default=DEFAULT_DATABASE)
+    parser.add_argument(
+        "--archive", dest="archive", help=archive_help, action="store_true"
+    )
+    parser.add_argument(
```

Comment:
We can just have `-archive`. When present we archive, when it isn't, we don't.

Reply:
Done:
https://github.com/debrief/pepys-import/pull/216/commits/a89b6de6c3fc129078b8acbcd2610cf87becdda8